docs/processing.md

```diff
@@ -313,5 +355,3 @@ There are some known issues and limitations that we hope to address while iterat
 - Handling array-typed attributes
 - Working on a array of points, rather than a single point
 - Metric alignment - for example defining an expression on two metrics, that may not be at the same timestamp
-- The collector has separate pipelines per signal - while the query language could apply cross-signal, we will need
```

Comment:
It's probably still worth keeping this line. It would be nice if eventually it could happen cross signal some day as some of our examples show :)

Reply:
Haha I'll add it back to keep the hope alive